### ISSUE TITLE
Unexpected mimetype in content-type: 'text/html'

### ISSUE THREAD

**Issue by garrettfoster13** [garrettfoster13/sccmhunter#40]
During registration in the HTTP module, Logan received an unexpected mime type error response.  This error has come up before but I haven't been able to reproduce it. I'm leaving this issue open so others can share their experience. 

I will be adding an update to the http module to output POST request reponses. If you're willing, please add your redacted results.

**Comment by Datacrass** [comment]
Had this happen a moment ago using Kerberos auth.

**Comment by Datacrass** [comment]
Verified. You are receiving a 403. Popped wireshark open at the same time to view the RAW HTTP traffic

**Comment by garrettfoster13** [comment]
@Datacrass my suspicion here is a PKI certificate requirement for registration. 

I'm going to try and create that environment in a lab this weekend and test. Thanks for the updates.

**Comment by qgrosperrin** [comment]
I'm also having that issue with password-based  authentication. It happened for me as the tool was trying to request the policy from management point.
```[15:11:44] INFO     [*] Attempting to grab policy from server.domain.local                                                                                                                                                              
[15:11:48] INFO     Unexpected mimetype in content-type: 'text/html'   ```

I'm running through SOCKS proxy

**Comment by garrettfoster13** [comment]
Closing due to fix in #64
